resources/windows/installer.nsi

```diff
@@ -52,27 +56,56 @@ Caption "${productName} Setup"
 SubCaption 3 " "
 SubCaption 4 " "
 
+;   --------------------------------
+;    Page layout
+;   --------------------------------
+
 Page custom welcome
+Page components
+Page directory
 Page instfiles
+Page custom finish finishEnd
+
+;   --------------------------------
+;    Control variables
+;   --------------------------------
 
 Var Image
 Var ImageHandle
 
+Var LaunchAppCheckbox
+Var LaunchAppCheckbox_State
+
+;   --------------------------------
+;    Installer init
+;   --------------------------------
+
 Function .onInit
 
     ; Extract banner image for welcome page
     InitPluginsDir
     ReserveFile "${banner}"
     File /oname=$PLUGINSDIR\banner.bmp "${banner}"
+    
+    ; Check if the application is currently running, show message if it is
+    retryInstallation:
+    FindWindow $0 "Chrome_WidgetWin_1" "Rocket.Chat"
```

Comment:
Is there some way to get the application name `Rocket.Chat` from parameter `productName`?